JumboCode/South-Florida-Jewish-Academy
# #96: Store real dates in DB for student birthdays

To test:
- run python3 resetDatabase.py to prime the database
- run all services
- open tab, navigate to "add student"
- add a student
- see robo3t, collection "student"
- each collection's DOB should have a Datetime object with a zero-ed out time
